src/components/Navigation/Navigation.scss

```diff
@@ -190,7 +190,7 @@ $disabled-fade: 0.6;
 }
 
 .SecondaryAction {
-  @include recolor-icon(var(--p-icon));
+  @include recolor-icon($fill-color: var(--p-icon));
```

Comment:
Switching to named vars throughout this PR for clarity. Should make things slightly easier to figure out what's going on when we remove `recolor-icon()`